### ISSUE TITLE
Google Play Store Developer Account Termination

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/learn-progressive-enhancement#30]
TIL: Google’s (“AI”) Decision making doesn’t care about your story.
https://blog.usejournal.com/google-wrongly-terminated-our-new-business-via-our-google-play-developer-account-5f5b7b742542
Via: 
https://news.ycombinator.com/item?id=19124324

**Comment by iteles** [comment]
Whoa, this is terrible!

**Comment by nelsonic** [comment]
@iteles yeah, really lame that rather than celebrating the launch of their App, 🎉> 😞 
they had to spend a month trying to _reason_ with the Google Machine. 🤖 
_Thankfully_ the App that we are building does not _need_ to be in the Play Store. 📱 
(_and we would never "outsource" the dev work to the lowest bidder on "UpWork" ..._) 🙄 
But the fact that Google has no process for _listening_ to _legitimate_ claims of unfair termination,
makes me _worried_ for the future of AI/Automation ...
If Google (_or any other Monopolistic company_) becomes ["The Law"](https://www.youtube.com/watch?v=itmNiTwHOsM) (_Judge, Jury & Executioner_)
The _world_ is in for a _really_ bad time. Not just "App Store" termination, but _real_ consequences!

Technology/platform _independence_ and breaking _free_ from vendor lock-in is _essential_!
As much as Google is "_useful_" now, (in)justice should not be decided by _any_ company.
